client/app/telecom/pack/xdsl/access/pack-xdsl-access.html

```diff
@@ -153,34 +163,31 @@ <h2 class="widget-presentation-title" data-translate="xdsl_details_title"></h2>
                         <div class="col-xs-12 col-md-4 service-title" data-translate="xdsl_details_ips_notifications"></div>
                         <div class="col-xs-12 col-md-8 service-item">
                             <a class="service-button"
-                               data-ui-sref="telecom.pack.xdsl.access-notifications"
-                               role="button">
-
+                               data-ui-sref="telecom.pack.xdsl.access-notifications">
                                 <div class="service-button-description">
                                     <div class="description-title-small description-title-fullHeight"
                                          data-ng-pluralize
                                          data-count="access.notificationsCount"
                                          data-when="{
                                             '0': '',
                                             'one': ('xdsl_details_notif_show_one' | translate : {count: access.notificationsCount}),
-                                            'other': ('xdsl_details_notif_show_many' | translate : {count: access.notificationsCount})
-                                         }">
+                                            'other': ('xdsl_details_notif_show_many' | translate : {count: access.notificationsCount})}">
                                     </div>
                                     <div data-ng-if="access.notificationsCount === 0">
                                         <span data-translate="xdsl_details_notif_show_none_part1"></span>
                                         <span data-translate="xdsl_details_notif_show_none_part2"></span>
                                     </div>
                                 </div>
 
-                                <i class="ovh-font ovh-font-arrow-right"></i>
+                                <span class="ovh-font ovh-font-arrow-right" aria-hidden="true"></span>
                             </a>
                         </div>
                     </div>
 
-                    <div class="row" data-ng-if="XdslAccess.packName==='sdsl'">
-                        <div class="col-xs-12 col-md-12">
-                            <a data-ui-sref="telecom.pack.xdsl.access-resiliation" data-translate="xdsl_resiliate" title="{{::'common_resiliation' | translate }}"></a>
-                        </div>
+                    <div data-ng-if="XdslAccess.packName==='sdsl'">
```

Comment:
Could you please use your `data-ng-if="XdslAccess.packName==='sdsl'"` directive directly on your `<a>` element.